### DISCUSSION TITLE
Accelerated Mobile Pages

### DISCUSSION THREAD

**Discussion by jac69** [dnnsoftware/Dnn.Platform#4766] [Q&A]
What is the DNN approach to the Accelerated Mobile Pages AMP?
Create alternate pages to the desktop ones with AMP?

Best
Jose

**Comment by bdukes** [comment]
Yes, there's no approach available to creating AMP compliant pages using DNN directly. You would need to read data out of DNN and generate AMP pages using some other mechanism to generate the HTML.
